documentation/src/main/wiki/SuiteReportFeatures.md

```diff
@@ -45,6 +45,18 @@ Moreover, every URL has information to which test group it belongs.
 
 ![Find example](assets/suiteReport/find-in-sidepanel-example.png)
 
+#### Browsing suite's history
+It is possible to view current suite's history by clicking current suite's name and version in the top toolbar. User can also navigate through versions by clicking next/previous arrow next to suite's version. 
+
+![Suite history](assets/suiteReport/suite-history.png)
+
+Clicking the suite's name in the top toolbar will show a popup in which the user can see every suite's version that is still available in the database. User can also directly go to the newest version of the suite by clicking 
```

Comment:
It looks like unfinished sentence :)

Reply:
Right, I don't know what happened there. Fixing now.